src/templates/app/user/security/account_closure.jsx

```diff
@@ -88,14 +88,10 @@ const AccountClosure = () => (
         <div id='logged_out' className='invisible'>
             <h1>{it.L('Account Closure')}</h1>
         </div>
-        <div id='msg_main' className='gr-gutter gr-padding-10 invisible'>
-            <h1 className='text-bold'>{it.L('Your account is now closed')}</h1>
+        <div id='msg_main' className='center-text gr-gutter gr-padding-30 invisible'>
             <p className='notice-msg'>
-                {it.L('You’ve successfully closed your account. We’ll send a confirmation email to [_1].', '<span id="current_email"></span>')}
+                {it.L('We\'re sorry to see you leave.[_1]Your account is now deactivated.', '<br />')}
```

Comment:
why not just have a real br tag and the second message on the line below it, instead of including it in translations :D